extras/nrc/str.sencoten/fix-wordlist.py

```diff
@@ -91,6 +105,22 @@ def uplus(c):
 
         wordlist[word] += count
 
-# Print as CRLF TSV:
-for word, count in wordlist.most_common():
-    print(word, count, sep='\t', end='\r\n')
+    # XXX: Artificially add ȻNEs
+    # It's not technically a single word according to the standard orthography
+    # e.g., *"ȻNEs YÁ¸" should be written "Ȼ NE SYA¸" ("that I go").
```

Comment:
This still adds "ȻNEs", but at a lower frequency than orthographically correct versions.